vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopVerifier.kt

```diff
@@ -95,6 +95,48 @@ class OidcSiopVerifier(
         )
     }
 
+    private val containerJwt = FormatContainerJwt(algorithms = arrayOf(JwsAlgorithm.ES256.identifier))
```

Comment:
Why is the algo hardcoded here?

Reply:
Fixed that in recent commits, for several usages of hardcoded algorithms in the OID4VCI/SIOP implementations.